src/app.js

```diff
@@ -1,5 +1,199 @@
+/* eslint-disable no-unused-expressions */
 const express = require('express');
 
 const app = express();
 
+app.use(express.json());
+
+// imports from lib
+
+const {
+  sayHello,
+  uppercase,
+  lowercase,
+  firstCharacters,
+  firstCharacter,
+} = require('./lib/strings');
+
+const { add, subtract, multiply, divide, remainder } = require('./lib/numbers');
+
+const { negate, truthiness, isOdd, startsWith } = require('./lib/booleans');
+
+const {
+  getNthElement,
+  arrayToCSVString,
+  addToArray2,
+  elementsStartingWithAVowel,
+  removeNthElement2,
+} = require('./lib/arrays');
+
+//  strings
+
+app.get('/', (req, res) => {
+  res.status(200).send('Hello, world!');
+});
+
+app.get('/strings/hello/:string', (req, res) => {
+  res.status(200).json({ result: sayHello(req.params.string) });
+});
+
+app.get('/strings/upper/:string', (req, res) => {
+  res.status(200).json({ result: uppercase(req.params.string) });
+});
+
+app.get('/strings/lower/:string', (req, res) => {
+  res.status(200).json({ result: lowercase(req.params.string) });
+});
+
+app.get('/strings/first-characters/:string', (req, res) => {
+  req.query.length
+    ? res.status(200).json({ result: firstCharacters(req.params.string, req.query.length) })
+    : res.status(200).json({ result: firstCharacter(req.params.string) });
+});
+
+// numbers
+
+app.get('/numbers/add/:num1/and/:num2', (req, res) => {
+  const int1 = parseInt(req.params.num1, 10);
+  const int2 = parseInt(req.params.num2, 10);
+
+  Number.isNaN(int1) || Number.isNaN(int2)
+    ? res.status(400).json({ error: 'Parameters must be valid numbers.' })
+    : res.status(200).json({ result: add(int1, int2) });
+});
+
+app.get('/numbers/subtract/:num1/from/:num2', (req, res) => {
+  const int1 = parseInt(req.params.num1, 10);
+  const int2 = parseInt(req.params.num2, 10);
+
+  Number.isNaN(int1) || Number.isNaN(int2)
+    ? res.status(400).json({ error: 'Parameters must be valid numbers.' })
+    : res.status(200).json({ result: subtract(int2, int1) });
```

Comment:
This is the third. I wont add full corrections for the others, but now you see whats causing them it should be an easy thing for you to fix:
```
  if (Number.isNaN(int1) || Number.isNaN(int2)) {
    res.status(400).json({ error: 'Parameters must be valid numbers.' });
  }
  res.status(200).json({ result: subtract(int2, int1) });
```